security/RememberLoginHash.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+class RememberLoginHash extends DataObject {
+
+	private static $db = array (
+		'DeviceID' => 'VarChar(40)',
+		'RememberLoginHash' => 'Varchar(160)',
+	);
+
+	private static $has_one = array (
+		'Member' => 'Member',
+	);
+
+	private static $indexes = array(
+		'DeviceID' => true
+	);
+
+	/**
+	 * Determines if logging out on one device also clears existing login tokens
+	 * on all other devices owned by the member.
+	 *
+	 * @config
+	 * @var bool
+	 */
+	private static $logout_across_devices = false;
```

Comment:
Needs to be `true` by default, otherwise I'd consider it "insecure by default" because you're relying on devs building a custom login management profile UI into their implementation. And the docs should make this crystal clear: If you set this to `false`, there's no way for users to revoke a login e.g. from a stolen device, unless you build one.